src/turtle/runner/Logging.d

```diff
@@ -56,7 +56,7 @@ public void decreaseLogIndent ( )
 
 *******************************************************************************/
 
-package void setupLogging ( )
+public void setupLogging ( )
```

Comment:
Probably could do with a release note mentioning this.

Reply:
Won't be picked because point releases ignore release notes. I will mention that manually when crafting notes.

Reply:
Ah yes. :+1: